drivers/overlay/overlay.go

```diff
@@ -812,7 +812,19 @@ func (d *Driver) getLowerDirs(id string) ([]string, error) {
 		for _, s := range strings.Split(string(lowers), ":") {
 			lower := d.dir(s)
 			lp, err := os.Readlink(lower)
-			if err != nil {
+			// if the link does not exist, we lost the symlinks during a sudden reboot.
```

Comment:
Nit, but could you resturcture this to be just 
```
if err != nil {
          if os.IsNotExist(err) {   
                   ...
          } else {
                  return nil, err
          }
}
```

Reply:
Yup, fixed